### PR TITLE
PMM-6462 New metrics for PG Stat Monitor.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: migrations/sql/07_pg_stat_monitor_columns.up.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ALTER TABLE metrics
+  ADD COLUMN `client_ip` LowCardinality(String),
```
Comment (askomorokhov) [inline migrations/sql/07_pg_stat_monitor_columns.up.sql:2]: I think this is duplicate column 
https://github.com/percona/qan-api2/blob/master/migrations/sql/01_init.up.sql#L8
> `client_host` LowCardinality(String) COMMENT 'client IP or hostname',

Comment (BupycHuk) [reply migrations/sql/07_pg_stat_monitor_columns.up.sql:2]: good point, thank you.

**Comment by askomorokhov** [comment]
@BupycHuk, please do `make gen` to update migration.

**Review by percona-csalguero — APPROVED** [review]

**Review by askomorokhov — CHANGES_REQUESTED** [review]
Overall looks good to me.
Please regen migrations.

**Review by JiriCtvrtka — APPROVED** [review]

**Comment by BupycHuk** [comment]
@AlekSi could you check why reviewdog is failing here? I tried key you shared to me in keybase twice, but it's still failing.

File: .travis.yml [hunk 2]
```diff
@@ -65,4 +65,5 @@ notifications:
...
+      # REVIEWDOG_GITHUB_API_TOKEN
+      secure: "Md83AEZnxZ/pXNEM4jrFO8vLVbWKW3SOzWqOCFFhsH/ojisQUF5SjMgH2ysZ/2J6AOps0w5+ny1ZAt0/Y2l64gX/5/qwbHERj/+f3kTKVXkGxOBYWCOdxdNlqYD90UJRG9HKqtcjNztd/JzoK+AZuE9rD9dKHOmmzLz/n6DsQEpu/jeDcn+rIrlRm7RrrKqNjbACnKEfMVL3WoSlOiKz6nrCq6Iyb9a3N1/UctaxSDR3nxZJ25E61C7tytFZkCx7uP1ep3Iok6uEdgpkPRmKUZUUOl1HR2O7dvi4HoNxWYut1xluhZ8Zbny/qOVJfBDnR5JRzgefzwKNTVUm97/NCKgaQcQXbNGieymErbi5bVnpFhcFxnqvrcpeT0drTw/ZTjD6bxpNnQLDXZeGUyEvLyEaNFreAI9LwADvzhpvi/xu1JoeLVGP4BfOlRj/v21PElS5kbK5FcPcEeTSqhJroOBe48p8dhlLI35U1/Y9KOBKS1Vlb117wpXlfFwMOhAKlXDdaUUGrQXwFb1nTAuo5WkPPgto0HBBK+mEtOUJLtEUZCfu4Zf1uH8PNzXSn6fhpSv36+Sklv+9obkqeJUCeEuoEJObhWrsVhGw/DcgzsRBmOtL+BcIpFDqYMKYrCAaOm+u2FIHGoCDE7ym2Xe+fw/K/XmSj/HsGcWNKkGZRso="
```
Comment (AlekSi) [inline .travis.yml:69]: @BupycHuk That's a Slack token. ReviewDog's is on line 17

Comment (BupycHuk) [reply .travis.yml:69]: Yeah, i got it right after sending a commen. Then removed it :blush:

**Pull request merged**